recipes/changeforest/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "changeforest" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e17b087297b9d3f5843bc37a7b44e55c229fe93c7c915a3f64a72da1e66ba911
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - pip
+    - maturin
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - changeforest
+  requires:
+    - python
+    - pip
+    - numpy
+    # - pytest
+  commands:
+    - pip check
+    # TODO: This depends of testdata, originally in changeforest-py/../testdata, now at
+    # changeforest-py/local_dependencies/changeforest/testdata. Make the testpath
+    # flexible in a future release.
+    # - pytest tests
```

This can be boiled down to the following
```suggestion
  requires:
    - pip
  commands:
    - pip check
```